documentation/working-with-testcafe-studio/creating-and-recording-tests.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: docs
+title: Creating and Recording Tests
+permalink: /documentation/working-with-testcafe-studio/creating-and-recording-tests
+---
+# Creating and Recording Tests
+
+This topic explains the basic concepts of tests in TestCafe Studio and describes how to create, record and modify them. The topic contains the following sections.
+
+* [Test Actions](#test-actions)
+* [Recording Tests](#recording-tests)
+    * [Record Configurations](#record-configurations)
+    * [Starting and Stopping Recording](#starting-and-stopping-recording)
+    * [During Recording](#during-recording)
+* [Creating Tests Manually](#creating-tests-manually)
+* [Modifying Tests](#modifying-tests)
+
+## Test Actions
+
+Tests consists of *test actions* to be performed with a tested page.
+
+TestCafe Studio provides a set of actions that you can use in tests. These actions are shown in the **Actions** panel on the right of the **Test Editor**. Actions are organized into a number of categories. For example, actions of the **On-Page Actions** category simulate user actions, like clicks, key presses, drags, etc., over your tested page. The actions of the **Assertions** category allow you to perform different verifications.
+
+IMAGE
+
+Each action has parameters that specify a target element and define how the action should be performed.
```

Comment:
Strictly speaking we have some actions without target element (`presskey`, `maximazeWindow` etc.)